### PR TITLE
tracee: make it the default binary

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does

This PR changes docker image entrypoint to use the new tracee binary. It removes the use of the older binaries `tracee-ebpf` and `tracee-rules`.

### 2. Explain how to test it

```
make -f builder/Makefile.tracee-container build-tracee
docker run  --name tracee --rm -it  --pid=host --cgroupns=host --privileged  -v /etc/os-release:/etc/os-release-host:ro      -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host  tracee:latest

# test different options
docker run  --name tracee --rm -it  --pid=host --cgroupns=host --privileged  -v /etc/os-release:/etc/os-release-host:ro      -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host  tracee:latest --output json

docker run  --name tracee --rm -it  --pid=host --cgroupns=host --privileged  -v /etc/os-release:/etc/os-release-host:ro      -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host  tracee:latest --filter set=signatures
```

Do the same for the full image. 

This is part of https://github.com/aquasecurity/tracee/issues/2355

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
Any new push will pick the new github testers (FYIO).

**Review by geyslan — APPROVED** [review]
LGTM overall.

Just some shellcheck caughts.

**Comment by yanivagman** [comment]
LGTM

**Review by yanivagman — APPROVED** [review]

File: builder/entrypoint.sh [hunk 1]
```diff
@@ -7,155 +7,66 @@
...
         --cache mem-cache-size=512 \
         --containers=${CONTAINERS_ENRICHMENT:="0"}\
         --capabilities bypass=${CAPABILITIES_BYPASS:="0"}\
         --capabilities add=${CAPABILITIES_ADD:=""}\
         --capabilities drop=${CAPABILITIES_DROP:=""}\
-        --filter event=${events} \
-        --output=out-file:${TRACEE_PIPE} &
-    tracee_ebpf_pid=$!
+        $@
+    #tracee_pid=$!
```
Comment (rafaeldtinoco) [inline builder/entrypoint.sh:41]: @josedonizetti nit: leftover here

Comment (josedonizetti) [reply builder/entrypoint.sh:41]: Thanks! Fixing it.

**Pull request merged**